src/ui/profile_preferences.ui

```diff
@@ -4145,5 +4162,38 @@ you can use it but there could be issues with aligning columns of text</string>
  <resources>
   <include location="../mudlet.qrc"/>
  </resources>
- <connections/>
+ <connections>
```

Comment:
:thinking: Whilst it is possible to define a signal-slot connection in the form/dialogue `.ui` file it is not so easy for some of us coders to spot/find them - speaking personally I would prefer it if these could be done via `connect(...)` calls in the `dlgProfilePrefences` constructor. Can I check @vadi2 if you share this view?

Reply:
I defined them in there but I can move it!